### PR TITLE
Hebrew translation updates

### PR DESCRIPTION
**Pull request by zvizvi**
- Hebrew translation updates

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

File: Files/MultilingualResources/Files.he-IL.xlf [hunk 1]
```diff
@@ -1679,7 +1679,7 @@
         </trans-unit>
         <trans-unit id="NavToolbarDisplayOptions.Text" translate="yes" xml:space="preserve">
           <source>Display Options</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">אפשרויות &amp;תצוגה</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">אפשרויות תצוגה</target>
```
Comment (BanCrash) [inline Files/MultilingualResources/Files.he-IL.xlf:1682]: It's fine to use needs review state when you want to leave a string so others (or you in the future) could check if it's okay if you are not totally confident about the result, but if you are confident about it the state should be translated. 

So, the question is, are the use of needs review intentional or should be changed to translated? This applies to all the needs review strings.

Comment (zvizvi) [reply Files/MultilingualResources/Files.he-IL.xlf:1682]: It is the right translation, so i will change the state to "translated".

Comment (BanCrash) [reply Files/MultilingualResources/Files.he-IL.xlf:1682]: > It is the right translation, so i will change the state to "translated".

Fine then! Don't forget to do the same on the rest if they are correct too:

Lines 1714, 1718, 1902, 1910, 1946, 2030 and 2042.

**Review by BanCrash — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**